bin/api-docs/gen-components-docs/markdown/props.js

```diff
@@ -0,0 +1,53 @@
+function renderPropType( type ) {
+	switch ( type.name ) {
+		case 'enum': {
```

Comment:
Any other prop types to handle?

Reply:
Maybe. I'll add more logic as I encounter them.